### PR TITLE
Add section about publishing with NativeScript CLI

### PR DESCRIPTION
**Pull request by Mitko-Kerezov**
Ping @PanayotCankov or @hdeshev  for a second opinion

### CODE REVIEW

**Comment by PanayotCankov** [comment]
It would be nice is you show an example in `submission automation` how the `tns` can be used in `CI` to submit. One that does not involve interactive CLI.

**Comment by Mitko-Kerezov** [comment]
@PanayotCankov good catch, updated with a some info and a couple of examples

File: core-concepts/publishing-ios-apps.md [hunk 1]
```diff
@@ -29,9 +29,10 @@ previous_url: /core-concepts/releasing-apps
   2. [Creating an App](#creating-an-app)
   3. [Builds](#builds)
     1. [Build Versioning](#build-versioning)
-    2. [Submit from Xcode](#submit-from-xcode)
-    3. [Submit with the Telerik AppManager](#submit-with-the-telerik-appmanager)
-    4. [Submission Automation](#submission-automation)
+    2. [Submit from NativeScript CLI](#submit-from-nativescript-cli)
```
Comment (ikoevska) [inline core-concepts/publishing-ios-apps.md]: Submit from the NativeScript CLI (fix section title below and anchor address)

File: core-concepts/publishing-ios-apps.md [hunk 2]
```diff
@@ -29,9 +29,10 @@ previous_url: /core-concepts/releasing-apps
...
+    3. [Submit from Xcode](#submit-from-xcode)
+    4. [Submit with the Telerik AppManager](#submit-with-the-telerik-appmanager)
```
Comment (ikoevska) [inline core-concepts/publishing-ios-apps.md]: Submit with Telerik AppManager (fix section title below and anchor address)

File: core-concepts/publishing-ios-apps.md [hunk 3]
```diff
@@ -303,6 +304,26 @@ export CFBundleVersion=`/usr/libexec/PlistBuddy app/App_Resources/iOS/Info.plist
 /usr/libexec/PlistBuddy app/App_Resources/iOS/Info.plist -c "Set :CFBundleVersion $CFBundleVersion.$BUILD_NUMBER"
 ```
 
+#### Submit from NativeScript CLI
+You can execute the following command using the CLI:
+```
+tns publish ios
+```
+
+The command will prompt for your `Apple ID` and `Password` for authentication with [iTunes  Connect](https://itunesconnect.apple.com) and then produce a `release` build and proceed to upload it to iTunes Connect.
```
Comment (ikoevska) [inline core-concepts/publishing-ios-apps.md]: Weird double spacing in `[iTunes  Connect]`

File: core-concepts/publishing-ios-apps.md [hunk 4]
```diff
@@ -303,6 +304,26 @@ export CFBundleVersion=`/usr/libexec/PlistBuddy app/App_Resources/iOS/Info.plist
...
+
+Alternatively you can run
```
Comment (ikoevska) [inline core-concepts/publishing-ios-apps.md]: Alternatively, you can use an existing build by running the following command:

File: core-concepts/publishing-ios-apps.md [hunk 5]
```diff
@@ -303,6 +304,26 @@ export CFBundleVersion=`/usr/libexec/PlistBuddy app/App_Resources/iOS/Info.plist
...
+```
+tns publish ios --ipa <Ipa File Path>
+```
+
+To use an already existing build.
```
Comment (ikoevska) [inline core-concepts/publishing-ios-apps.md]: Remove

File: core-concepts/publishing-ios-apps.md [hunk 6]
```diff
@@ -303,6 +304,26 @@ export CFBundleVersion=`/usr/libexec/PlistBuddy app/App_Resources/iOS/Info.plist
 /usr/libexec/PlistBuddy app/App_Resources/iOS/Info.plist -c "Set :CFBundleVersion $CFBundleVersion.$BUILD_NUMBER"
 ```
 
+#### Submit from NativeScript CLI
+You can execute the following command using the CLI:
```
Comment (ikoevska) [inline core-concepts/publishing-ios-apps.md]: Some context needed here - where should the command be executed - in the project dir? Or somewhere else?

File: core-concepts/publishing-ios-apps.md [hunk 7]
```diff
@@ -303,6 +304,26 @@ export CFBundleVersion=`/usr/libexec/PlistBuddy app/App_Resources/iOS/Info.plist
...
+The command will prompt for your `Apple ID` and `Password` for authentication with [iTunes  Connect](https://itunesconnect.apple.com) and then produce a `release` build and proceed to upload it to iTunes Connect.
+
+Alternatively you can run
+```
+tns publish ios --ipa <Ipa File Path>
+```
+
+To use an already existing build.
+
+For more information you can run
```
Comment (ikoevska) [inline core-concepts/publishing-ios-apps.md]: For more information, run the following command:

File: core-concepts/publishing-ios-apps.md [hunk 8]
```diff
@@ -338,6 +359,20 @@ as well as support for cloud builds and *Google Play* submission.
 Read more about app submission process in the *Telerik Platform* in ['Introduction to Telerik AppManager'](http://docs.telerik.com/platform/appmanager/getting-started/introduction)
 
 #### Submission Automation
+Automation can be achieved using the NativeScript CLI only. All of the parameters needed for publishing can be passed to the `publish` command directly:
+```
```
Comment (ikoevska) [inline core-concepts/publishing-ios-apps.md:362]: Consider adding an empty line between the code block and the opening paragraph.

File: core-concepts/publishing-ios-apps.md [hunk 9]
```diff
@@ -338,6 +359,20 @@ as well as support for cloud builds and *Google Play* submission.
...
+tns publish ios [<Apple ID> [<Password> [<Mobile Provisioning Profile Identifier> [<Code Sign Identity>]]]]]
+```
+For example, assuming that you want to issue a build using a mobile provision with an identifier *d5d40f61-b303-4fc8-aea3-fbb229a8171c*, you could run
```
Comment (ikoevska) [inline core-concepts/publishing-ios-apps.md]: Colon (:) at the end of the statement

File: core-concepts/publishing-ios-apps.md [hunk 10]
```diff
@@ -338,6 +359,20 @@ as well as support for cloud builds and *Google Play* submission.
...
+```
+tns publish ios my-apple-id my-password d5d40f61-b303-4fc8-aea3-fbb229a8171c "iPhone Distribution"
+```
+Note that the `Code Sign Identity` can be set to something generic like *iPhone Distribution* in order to let the build automatically detect a code sign identity.
+Of course you could also use
```
Comment (ikoevska) [inline core-concepts/publishing-ios-apps.md]: Insert an empty line between the two paragraphs. Then, rework this statement to "You can also automate the uploads of already built packages:"

**Comment by Mitko-Kerezov** [comment]
Ping @ikoevska with fixed comments

**Comment by ikoevska** [comment]
:+1:

**Pull request merged**